tests/acceptance/docker/config/hashicorp/hashicorp.go

```diff
@@ -1,23 +1,17 @@
 package hashicorp
 
-import (
-	"fmt"
-	"io"
-	"net/http"
-	"os"
-)
-
-const defaultHashicorpVaultImage = "library/vault:1.8.2"
+const defaultHashicorpVaultImage = "consensys/quorum-hashicorp-vault-plugin:v1.1.4"
 const defaultHostPort = "8200"
 const defaultRootToken = "myRoot"
 const defaultHost = "localhost"
+const defaultMountPath = "orchestrate"
```

Comment:
why `orchestrate` ? Also this mountPath is only for the plugin right? Could be called `pluginMountPath` ?

Reply:
Updated